### PR TITLE
Basic layout + nav :boat: 

### LINKED ISSUE #19: Create user component in navbar
image

there should not be logic in this component. just prop for everything. we'll do the redirection to profile later you can skip that

### LINKED ISSUE #22: Create basic app layout

### PR DESCRIPTION
**Pull request by alexandrelam**
basic layout + nav
image

### CODE REVIEW

**Review by RomainGuarinoni — CHANGES_REQUESTED** [review]
Everything is great. Just when I click on either "Musiciens", "Groupes" or "Evènements", the app do not redirect to the login page if the user is not logged in. The redirection seems to only work when we click on the Profile button
error

**Comment by RomainGuarinoni** [comment]
I think the layout shouldn't display the navbar and the user profil when the routes are `/login` or `/register`, what do you think ?

File: components/Navbar.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        <ITelescope /> Find a musician
+      </span>
+      <nav className="flex flex-col gap-8">
+        <NavItem text="Musiciens" href="/musician">
+          <IMusicNote />
+        </NavItem>
+        <NavItem text="Groupes" href="/groups">
+          <IPeople />
+        </NavItem>
+        <NavItem text="Evènements" href="/events">
```
Comment (Meltical) [inline components/Navbar.tsx]: `<NavItem icon={<ICalendar  />} text="Evènements" href="/events">`

**Review by ThomasBernard0 — APPROVED** [review]

**Comment by alexandrelam** [comment]
> Everything is great. Just when I click on either "Musiciens", "Groupes" or "Evènements", the app do not redirect to the login page if the user is not logged in. The redirection seems to only work when we click on the Profile button error

fixed [823a85f](https://github.com/Find-A-Musician/findAMusicianFrontend/pull/23/commits/823a85f2fe4408e5372d0bef386267f335a71742)

**Pull request merged**